app/helpers/moments_helper.rb

```diff
@@ -99,6 +99,35 @@ def moment_form_inputs
   end
   # rubocop:enable MethodLength
 
+  # rubocop:disable MethodLength
+  def moments_stats
+    total_count = current_user.moments.all.count
```

Comment:
This is a good opportunity to reduce the size of this method and put some of the logic into private methods in this helper file.

Reply:
We also want to move the tests related to this code (in application_controller_spec.rb) into moments_helper_spec.rb (which unfortunately doesn't exist yet)